.travis.yml

```diff
@@ -1,10 +1,6 @@
-sudo: required
-dist: trusty
-
-env:
-  global:
-    - DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
-    - NUGET_XMLDOC_MODE=skip
+language: csharp
+mono: none
+dotnet: 2.0.0
```

Comment:
Is this new? I've not seen it before.

Reply:
Hmmm, not sure. Anyway, I've reverted it because it's slower, and the current solution is something we can control and has an appropriate level of complexity.